EXAMPLE/knx_iot_wakeful_main.c

```diff
@@ -308,6 +308,39 @@ int main(void)
 				 sn[5]);
 		app_set_serial_number(serial_number_str);
 	}
+	
+	char pwd[33];
+	error = knx_get_stored_password(pwd);
+	if (error)
+	{
+		PRINT_APP("Error: Stored password not found! Using default value...\n");
+		PRINT_APP(
+			"Please create the data file using knx-gen-data and flash it with chilictl in order to fix this issue.\n");
+	}
+	else
+	{
+		oc_spake_set_password(pwd);
+	}
+	
+	uint8_t salt[32], rand[32];
+	uint32_t it;
+	mbedtls_mpi w0;
+	mbedtls_ecp_point L;
+	mbedtls_mpi_init(&w0);
+	mbedtls_ecp_point_init(&L);
+	error = knx_get_stored_spake(salt, rand, &it, &w0, &L);
+	if (error)
```

Comment:
I think you probably also want to free `w0` and `L` in the error case. (Likewise for sleepy_main.c)

Reply:
This is already handled in the SDK - any error cases dealloc anything used or returned. I can still add it in though if you'd prefer

Reply:
Ah I see, that's good enough then, no need to add it here.